pkg/controller/generic_control.go

```diff
@@ -241,7 +241,6 @@ func (w *typedWrapper) CreateOrUpdateServiceAccount(controller client.Object, sa
 	}
 	return result.(*corev1.ServiceAccount), err
 }
-
```

Comment:
Please revert this.

Reply:
This makes tests work, refer to the above discussions.

Reply:
done